docs/data/toolpad/getting-started/quickstart.md

```diff
@@ -0,0 +1,129 @@
+# Quickstart
+
+<p class="description">Get started building a Toolpad app.</p>
+
+This page will guide you into creating your very first Toolpad application. You will create a "hello world" type of application that will ask for a name and print a greeting.
+
```

Comment:
I would suggest adding this section to give the reader more info about what they'll learn here.

```suggestion

## Purpose

This Quickstart guide is intended to introduce you to the fundamentals of building with Toolpad.
By the end, you should be able to:

- set up a new Toolpad app
- navigate through your workspace
- add components, data sources, and queries to an app
- bind data sources and components
- deploy a Toolpad app
```